crates/types/src/partition_table.rs

```diff
@@ -497,7 +506,11 @@ impl TryFrom<PartitionTableShadow> for PartitionTable {
     fn try_from(value: PartitionTableShadow) -> Result<Self, Self::Error> {
         let mut builder = PartitionTableBuilder::new(value.version);
         // replication strategy is unset if data has been written with version <= v1.1.3
-        builder.set_partition_replication(value.replication.unwrap_or_default());
+        builder.set_partition_replication(
+            value
+                .replication
+                .unwrap_or(PartitionReplication::Everywhere),
```

Comment:
I am not sure if this is the right value to set in this case maybe a replication property of 1 is more appropriate

Reply:
It is probably also fine to set it to `PartitionReplication::Limit(1)`. This would align the upgrade path from 1.1.3 to get away from the `Everywhere` replication.